proxy/handler.go

```diff
@@ -0,0 +1,52 @@
+package proxy
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/ONSdigital/dp-legacy-cache-proxy/config"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+func (proxy *Proxy) Handler(ctx context.Context, w http.ResponseWriter, req *http.Request, cfg *config.Config) {
+	targetURL := cfg.BabbageURL + req.URL.String()
+
+	client := &http.Client{}
```

Comment:
I'd move this declaration closer to where it's being used (above `client.Do(proxyReq)`). For two reasons:
- Them being together makes the code easier to read.
- The next block does not use the client, so if `err` is not `nil`, then we would have created this variable unnecessarily.